vfs/dir.go

```diff
@@ -916,6 +918,17 @@ func (d *Dir) cachedNode(relativePath string) Node {
 	return node
 }
 
+// Update the object but don't update the directory cache - for use by
```

Comment:
I think this comment is wrong. It is certainly confusing me!

I think you mean something like this?

```go
// Update the directory in the directory cache with a new fs.Directory.
// For use by the directory cache.
// Does not update the VFS cache.
```